### PR TITLE
DEV-80392 changes

### PR DESCRIPTION
**Pull request by gagansingh355**

### CODE REVIEW

File: code/parse_test.go [hunk 1]
```diff
@@ -410,7 +410,7 @@ func TestParseKinesisFirehoseLogs(t *testing.T) {
 	logs := parseCloudWatchLogs(cloudWatchEvent)
 
 	localTime := time.Local
-	timeValue := time.Date(2021, time.April, 26, 11, 15, 15, 22800000, time.Local)
+	timeValue := time.Date(2021, time.April, 26, 11, 15, 15, 228000000, time.Local)
```
Comment (ahsan13jan) [inline code/parse_test.go:413]: What is this change? 
I remember we had discussion that tests where failing if we create time from string. You can also try to convert it to UTC and try.

File: code/parse_test.go [hunk 2]
```diff
@@ -519,3 +519,27 @@ func TestELBlowLogs(t *testing.T) {
...
+			Data: "H4sIAAAAAAAAAG2Qy2rDMBBFf8VoHWO9LGmyM9QNXbRdOLsSiiLLwTR+ICk1JeTfO2kJbaEzm+HO5czjTAYfoz347cfsyZrcVdvq9bFummpTkxWZltEHlJk0RoLSwKhG+TgdNmE6zdgp7BKL0MZi9GmZwtvDmHzorPPftiYFbwf0+bHPKTjlulYp7oTpWtnm1jk/J7TG0z660M+pn8b7/oiMSNYvxJLdF6Z+92O6KmfSt0gTihsGXHKGMKpZKTiDUgKlnDHQknIhQVAqFIDkoI00TJTK4KTU48XJDrg8U1yB0LqkGKvbJxDPs98HZ//unuV/8od1Kw0F1J+ery8ll93lE2T4ipxrAQAA",
+		},
+	}
+
+	logs := parseCloudWatchLogs(cloudWatchEvent)
+
+	localTime := time.Local
+	timeValue := time.Date(2021, time.July, 22, 12, 39, 10, 000000000, time.Local)
+	if strings.Contains(localTime.String(), "UTC") { //Test case is running at system with time.Local as UTC
+		timeValue = time.Date(2021, time.July, 22, 07, 9, 10, 000000000, time.Local)
```
Comment (ahsan13jan) [inline code/parse_test.go:535]: can try time.UTC() will always have it in UTC.

**Review by ahsan13jan — CHANGES_REQUESTED** [review]
Try using UTC time. If that makes time creation more readable.

File: code/parse_test.go (hunk 2 above)
Comment (gagansingh355) [reply code/parse_test.go:535]: I have tried with this method but this method fails the test cases because it does not contain all the fields.

**Review by ahsan13jan — APPROVED** [review]

**Pull request merged**